spec/frameSpec.js

```diff
@@ -0,0 +1,29 @@
+describe('Frame', () => {
+  let frame;
+
+  beforeEach( () => { frame = new Frame; });
+
+  it('ends the turn when a strike is rolled and makes bonus = 2', () => {
+    frame.addTurn(10);
+    expect(frame.inTurn).toEqual(false);
```

Comment:
addTurn isnt maybe the best phrase for this. Gets a bit confusing but cant think of a better alternative right now.